### PR TITLE
add missing LICENSE

### PR DESCRIPTION
**Pull request by smlng**
see title

### CODE REVIEW

File: LICENSE [hunk 1]
```diff
@@ -0,0 +1,504 @@
...
+                       Version 2.1, February 1999
+
+ Copyright (C) 1991, 1999 Free Software Foundation, Inc.
+ 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301  USA
+ Everyone is permitted to copy and distribute verbatim copies
+ of this license document, but changing it is not allowed.
+
+[This is the first released version of the Lesser GPL.  It also counts
+ as the successor of the GNU Library Public License, version 2, hence
+ the version number 2.1.]
```
Comment (miri64) [inline LICENSE]: We don't have this section in the [LICENSE file of RIOT's main code](https://github.com/RIOT-OS/RIOT/blob/master/LICENSE#L9).

File: LICENSE [hunk 2]
```diff
@@ -0,0 +1,504 @@
...
+CONSEQUENTIAL DAMAGES ARISING OUT OF THE USE OR INABILITY TO USE THE
+LIBRARY (INCLUDING BUT NOT LIMITED TO LOSS OF DATA OR DATA BEING
+RENDERED INACCURATE OR LOSSES SUSTAINED BY YOU OR THIRD PARTIES OR A
+FAILURE OF THE LIBRARY TO OPERATE WITH ANY OTHER SOFTWARE), EVEN IF
+SUCH HOLDER OR OTHER PARTY HAS BEEN ADVISED OF THE POSSIBILITY OF SUCH
+DAMAGES.
+
+                     END OF TERMS AND CONDITIONS
+
+           How to Apply These Terms to Your New Libraries
```
Comment (miri64) [inline LICENSE]: This part neither. The file in RIOT-OS/RIOT [just ends after "END OF TERMS AND CONDITIONS"](https://github.com/RIOT-OS/RIOT/blob/master/LICENSE#L454).

Also, I'm pretty sure we don't apply the terms to our files the way it is described below.

**Comment by smlng** [comment]
@miri64 I simply used the add licences feature under insights->community and clicked on LGPL-2.1 which adds the standard text for that license.

Does it make sense the change the LICENSE text from the original? Maybe we should rather update the RIOT one.

**Comment by miri64** [comment]
> @miri64 I simply used the add licences feature under insights->community and clicked on LGPL-2.1 which adds the standard text for that license.

I was suspecting something like that, just wanted to make sure that we don't get some random LGPL-2.1 copy-pasta



> Does it make sense the change the LICENSE text from the original? Maybe we should rather update the RIOT one.

I don't think we have to GitHub recognizes the RIOT one as LGPL v2.1 and I believe there is nothing to be achieved by changing the LICENSE file just for some flavor-text.

**Pull request merged**